### ISSUE TITLE
Add static pages

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#2783]
Related to #2748, we need new static pages on the frontend. Should be easy!

- [ ] About
- [ ] Rating Policy

**Comment by tsl143** [comment]
Can I take this?

**Comment by tofumatt** [comment]
Sure thing, but my apologies for not giving a lot of context. We don't actually have designs for these pages yet, although I guess the 404-type page is enough to start with.

If you copied the content for these pages from addons-server that would be great. Let me know if that makes sense.

**Comment by tsl143** [comment]
Yes I will try to replicate the about and rating page with 404 as template and content from addons-server pages.

**Comment by tsl143** [comment]
Hi @tofumatt ,
I have created `about` page. But I am stuck with `<Link>` component,
I cant figure out how to `dangerouslysetInnerHtml` with `i18n.sprintf(i18n.gettext())`
See https://github.com/tsl143/addons-frontend/blob/I2783/src/amo/components/StaticPages/About/index.js#L25
This is not rendering as expected. The link doesnt get displayed as anchor tag

**Comment by tofumatt** [comment]
You shouldn't need to be using dangerouslySetInmerHTML that often, and in those cases you don't need to. You can put the Link components directly in the JSX.

Also, Link is not a tag but a react component and dangerouslySetInnerHTML will not render JSX, it will only output raw HTML as supplied. So you can't put JSX in a string then have it render inside dangerouslySetInnerHTML. It will only output basic HTML.

You may want to look around other components in the app to see how a similar case is handled, I think we have some. :-)

Matthew Riley MacPherson

**Comment by tsl143** [comment]
Hi @tofumatt , I updated the code without using dangerouslySetInmerHTML, but I am afraid, will it not be hassle for `Locale` strings, in addition We still have 404 links on the page as `FAQ`.
https://github.com/tsl143/addons-frontend/blob/9631b9324f14aab7e09b0689423945bf289ae692/src/amo/components/StaticPages/About/index.js

Also I am unable to find Rating policy page.

**Comment by tsl143** [comment]
ping @tofumatt

**Comment by tsl143** [comment]
Finally this is done :)
@muffinresearch , shall I create FAQ page?

**Comment by ValentinaPC** [comment]
Are these 2 pages public?

**Comment by ValentinaPC** [comment]
> Related to #2748, we need new static pages on the frontend. Should be easy!
>     About
>     Rating Policy

Are these 2 pages public?

**Comment by tofumatt** [comment]
To appear on the front end they will need new routes in nginx. This is an ops bug, as is any new route for the frontend. A new issue assigned to ops should be filed, I guess.

Matthew Riley MacPherson

**Comment by muffinresearch** [comment]
QA'ing this is blocked by https://github.com/mozilla/addons-frontend/issues/3409

**Comment by muffinresearch** [comment]
> @muffinresearch , shall I create FAQ page?

There's no need - that page was dropped.

**Comment by ValentinaPC** [comment]
Mark as verified based on https://github.com/mozilla/addons-frontend/issues/3409 .
